src/riak_cs_wm_common.erl

```diff
@@ -472,6 +475,10 @@ post_authentication({error, bad_auth}, RD, Ctx, _, _) ->
     %% given keyid was found, but signature didn't match
     _ = lager:debug("bad_auth"),
     riak_cs_wm_utils:deny_access(RD, Ctx);
+post_authentication({error, reqtime_tooskewed} = Error, RD,
+                    #context{response_module = ResponseMod} = Ctx, _, false) ->
+    _ = lager:debug("reqtime_tooskewed"),
+    ResponseMod:api_error(Error, RD, Ctx);
 post_authentication({error, {auth_not_supported, AuthType}}, RD,
                     #context{response_module=ResponseMod} = Ctx, _, _) ->
     %% given keyid was found, but signature didn't match
```

Comment:
Not for this PR, but would you please remove this comment line? It is simply wrong.